validator/entrypoint.sh

```diff
@@ -6,18 +6,19 @@ WEB3SIGNER_API="http://web3signer.web3signer-${NETWORK}.dappnode:9000"
 
 # MEVBOOST: https://docs.teku.consensys.net/en/latest/HowTo/Builder-Network/
 if [ -n "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" ] && [ "$_DAPPNODE_GLOBAL_MEVBOOST_PRATER" == "true" ]; then
-  echo "MEVBOOST is enabled"
+  echo "MEV-BOOST is enabled"
   MEVBOOST_URL="http://mev-boost.mev-boost-goerli.dappnode:18550"
+  EXTRA_OPTS="--validators-builder-registration-default-enabled=true ${EXTRA_OPTS}"
```

Comment:
I have never had it happen to me personally, but I've seen many support cases with Teku MEV-Boost issues, its not common but ive had too many cases reported of users getting MEV-Boost Errors about this flag `--validators-proposer-blinded-blocks-enabled=true` not being enabled explicitly.  I know the flag already in the code enabling default registration is supposed to set the blinded block flag automatically but ive just seen too many reports of errors saying mev -boost errors due to the blinded blocks flag not being enabled, so unless having it passed explcitly is known to cause issues, i think it should just be added here in this line